api_docs/changelog.md

```diff
@@ -204,9 +204,12 @@ No changes; feature level used for Zulip 8.0 release.
 **Feature level 224**
 
 * [`GET /events`](/api/get-events), [`GET /messages`](/api/get-messages),
-  [`GET /messages/{message_id}`](/api/get-message): The `wildcard_mentioned`
-  flag was deprecated, replaced with `stream_wildcard_mentioned` and
-  `topic_wildcard_mentioned`, but it is still available for backwards compatibility.
+  [`GET /messages/{message_id}`](/api/get-message),
+  [`POST /messages/flags`](/api/update-message-flags),
+  [`POST /messages/flags/narrow`](/api/update-message-flags-for-narrow):
+  The `wildcard_mentioned` flag was deprecated and replaced with the
+  `stream_wildcard_mentioned` and `topic_wildcard_mentioned` flags, but
+  it is still available for backwards compatibility.
```

Comment:
I don't think the personal update flag endpoints can interact with this flag; it's part of `NON_EDITABLE_FLAGS` and thus not editable.

Reply:
Hmm, and I guess the API documentation for that endpoint actually has a bunch of similar things listed that are not to be edited.

Reply:
Should the 'Available flags' table possibly be moved to a separate, non-endpoint page... with a different title? Or maybe the changelog entry links to that table and not the endpoints. Something similar to what we did for feature levels 194 and 177?

I see what you mean in regards to these endpoints not being able to update those particular message flags via the `flags` parameter. So maybe the table's title needs to be changed either way.

Reply:
How about this ...

![Screenshot from 2024-03-13 16-53-29](https://github.com/zulip/zulip/assets/63245456/4ae9fa26-b2ad-4e06-8ec7-573d544f603c)

Reply:
Yeah that seems fine.